pyqtgraph/widgets/SpinBox.py

```diff
@@ -77,7 +77,6 @@ def __init__(self, parent=None, value=0.0, **kwargs):
             'step': D('0.01'),  ## if 'dec' is false, the spinBox steps by 'step' every time
                                 ## if 'dec' is True, the step size is relative to the value
                                 ## 'step' needs to be an integral divisor of ten, ie 'step'*n=10 for some integer value of n (but only if dec is True)
-            'log': False,   # deprecated
```

Comment:
Is there a reason this one is not warned like the others?

Reply:
It would only raise an Exception

```python
            if self.opts['log']:	
                raise Exception("Log mode no longer supported.")
```

Figure I'm just removing reference in the documentation, I'm not actually removing any functionality.